pkg/kubelet/dockershim/metrics/metrics.go

```diff
@@ -132,14 +143,13 @@ var registerMetrics sync.Once
 // Register all metrics.
 func Register() {
 	registerMetrics.Do(func() {
-		prometheus.MustRegister(DockerOperationsLatency)
-		prometheus.MustRegister(DockerOperations)
-		prometheus.MustRegister(DockerOperationsErrors)
-		prometheus.MustRegister(DockerOperationsTimeout)
-		prometheus.MustRegister(DeprecatedDockerOperationsLatency)
-		prometheus.MustRegister(DeprecatedDockerOperations)
-		prometheus.MustRegister(DeprecatedDockerOperationsErrors)
-		prometheus.MustRegister(DeprecatedDockerOperationsTimeout)
```

Comment:
Is there a reason we are not registering this anymore?

Reply:
It’s registered in the lines below.

Reply:
I expanded the rest of the file, and still don't see `DeprecatedDockerOperationsTimeout` registered anywhere...  can you link to it?

Reply:
ah, good catch. super odd it's not there, as i did a find and replace.